### PR TITLE
Retrieve and show renewal estimate for subscription

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Retrieve and show renewal estimate for subscription

Use new api for this

Show totals even if 0

[stage-12]

## Motivation and Context
Fix for #2658 

## How Has This Been Tested?
Tested locally with the updated store server version. Will update unit tests in a subsequent PR.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

**Comment by alex-deaconu** [comment]
The response will no longer appear correctly because the API has been updated. `[stage-13]` for https://github.com/Rise-Vision/rise-vision-apps/pull/2684 will work correctly. Thanks.

File: src/scripts/billing/services/svc-subscription-factory.js [hunk 1]
```diff
@@ -66,6 +70,15 @@ angular.module('risevision.apps.billing.services')
             factory.item = resp.item;
 
             _updatePaymentSourceId();
+
+            if (estimateRenewal) {
+              return billing.estimateSubscriptionRenewal(subscriptionId);
```
Comment (ezequielc) [inline src/scripts/billing/services/svc-subscription-factory.js:75]: @alex-deaconu Could we call this in parallel with billing.getSubscription() above? Thanks!

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Looks good 👍 
One question/suggestion inline. Thanks

File: src/scripts/billing/services/svc-subscription-factory.js (hunk 1 above)
Comment (alex-deaconu) [reply src/scripts/billing/services/svc-subscription-factory.js:75]: @ezequielc very good suggestion. In my original prototype I was using information from the getSubscription call but that's no longer the case. I will update in a subsequent PR. Thanks.

**Pull request merged**